tests/unit/providers/test_cerebras_provider.py

```
@@ -1,29 +1,44 @@
 
 
 @pytest.mark.asyncio
-async def test_stream_with_response_format_raises() -> None:
-    api_key = "test-api-key"
-    model = "model-id"
-    messages = [{"role": "user", "content": "Hello"}]
+async def test_stream_with_response_format_passes_to_completion_create() -> None:
+    response_format = {"type": "json_object"}
 
-    provider = CerebrasProvider(api_key=api_key)
+    async def _stream() -> Any:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_stream`

(ANN401)